src/EventStore.Core/TransactionLog/Chunks/TFChunk/TFChunk.cs

```diff
@@ -138,6 +145,10 @@ private enum CacheStatus {
 
 		private IChunkReadSide _readSide;
 
+		private IChunkTransform _transform;
+		private ReadOnlyMemory<byte> _transformHeader;
+		private readonly IChunkTransform _identityTransform = new IdentityChunkTransform();
```

Comment:
we can probably have a static `IdentityChunkReadTransform` and share it among the chunks rather than instantiating the IdentityChunkTransform, IdentityChunkReadTransform and IdentityChunkWriteTransform for each chunk

Reply:
great idea :+1: